src/Faker/Provider/ar_EG/Internet.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace Faker\Provider\ar_EG;
+
+class Internet extends \Faker\Provider\Internet
+{
+    protected static $userNameFormats = [
+        '{{lastNameAscii}}.{{firstNameAscii}}',
+        '{{firstNameAscii}}.{{lastNameAscii}}',
+        '{{firstNameAscii}}##',
+        '?{{lastNameAscii}}',
+    ];
+    protected static $safeEmailTld = [
+        'com', 'com.eg', 'eg', 'me', 'net', 'org',
+    ];
+
+    protected static $tld = [
+        'biz', 'com', 'come.eg', 'info', 'eg', 'net', 'org',
+    ];
+
+    protected static $lastNameAscii = [
+        'ahmed',
+        'mostafa',
+        'mahmoud',
+        'carmen',
+        'rakeen',
+        'hazem',
+        'ezz',
+        'hemeida',
+        'ramah',
+        'fahmy',
+        'ehab',
+        'karim',
+        'abdulaziz',
+        'elsherbiny',
+        'karam',
+        'abdulaziz',
+        'bayoumi',
+        'tharwat',
+        'elshamy',
+        'youssef',
+        'rizk',
+        'ramzy',
+        'younes',
+        'selim',
+    ];
+    protected static $firstNameAscii = [
```

Comment:
Any source for these arrays?

Reply:
> 

a literal translation for some of the people names from Person.php